proposed/http-client/http-client-meta.md

```diff
@@ -23,24 +23,23 @@ clients will allow libraries to be decoupled from specific implementation such a
 
 ### Non-Goals
 
-* Support for asynchronous HTTP requests is left for another future PSR;
+* Support for asynchronous HTTP requests is left for another future PSR.
 * This PSR does not define how to configure an HTTP client. It only specifies the
-  default behaviours;
-* This PSR is neutral about the use of middlewares (PSR-15).
+  default behaviours.
+* This PSR is neutral about the use of middleware.
```

Comment:
PSR-15 addresses server-side middleware, so it's irrelevant to reference it here.

Also, the plural of middleware is middleware. :)